### DISCUSSION TITLE
Display filters applied on the list itself

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#4223] [Feature Requests & Ideas]
When viewing a List with filters on, there are currently no indicators that a filter is being applied or the list of filter options being applied.

This should be displayed more prominently on the List page.

**Comment by yfi** [comment]
I agree. The filter rules bar is quite big and pushes the list down out of view. Especially with more complex filters that have 3+ rows. And it is possible to minimize the filter rules bar while filters are still being applied on the list. At this point there is no visual cue that filters are applied and list rows seem to be missing. 

Some ideas:
1. The original suggestion of displaying a compact summary of filters on the list component itself is a great solution.

2. Separate the filter toggle from the search bar and move it to it's own bar. 
This bar would expand to define filter rules as is, and when minimized it could display a summary of applied filters.

3. Give the filter toggle a more "active" appearance when filters are applied. So when filters are applied, and the filter rules bar is minimized the filter pill is highlighted and shows the filter count... ie. Filters (3)

**Reply by zubricks** [reply]
Hey @yfi — Appreciate the feedback, could you post it [here](https://github.com/payloadcms/payload/discussions/4193) instead? Thank you.
